besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1255,6 +1260,8 @@ private void addSubCommands(
         RLPSubCommand.COMMAND_NAME, new RLPSubCommand(resultHandler.out(), in));
     commandLine.addSubcommand(
         OperatorSubCommand.COMMAND_NAME, new OperatorSubCommand(resultHandler.out()));
+    commandLine.addSubcommand(
+        ValidateConfigSubCommand.COMMAND_NAME, new ValidateConfigSubCommand(resultHandler.out()));
```

Comment:
Can you not inject it? 

`new ValidateConfigSubCommand(commandLine, resultHandler.out()));`

Reply:
yes. yes I can